docs/installation.rst

```diff
@@ -154,6 +157,20 @@ the :ref:`devnotes` will provide
 you with some useful pointers to get started.
 
 
+.. _optional-dependencies:
+
+Optional dependencies for analysis features
+-------------------------------------------
+
```

Comment:
From my experience users rarely read such a block text in the docs. I think it's wise to first show a list of "optional packages" along with a code block listing the conda or pip command to install them. You can then add the more detailed info in a paragraph afterwards.

Reply:
Sounds good to be.